resources/js/components/FormField.vue

```diff
@@ -63,21 +93,26 @@ export default {
 		},
 
 		/**
-         * Fill the given FormData object with the field's internal value.
-         */
+		 * Fill the given FormData object with the field's internal value.
+		 */
 		fill(formData) {
 			formData.append(this.field.attribute, this.value || '')
 		},
 
 		/**
-         * Update the field's internal value.
-         */
+		 * Update the field's internal value.
+		 */
 		handleChange(value) {
 			this.value = value
 		},
 
 		loadOptions () {
-			window.Nova.request().get(this.field.url).then(({data}) => {
+			let url = this.field.url;
+			if(this.parentVal) {
+				url = url + '?' + this.field.parent_field + '=' + this.parentVal;
```

Comment:
Just a quickie mate to future proof it, can we build up a `params` object instead and pass it to the `Nova.request.get(url, {params})` please?

Reply:
I tried passing params to nova get but it wouldn't work :/